rdr_service/api/nph_intake_api.py

```diff
@@ -248,6 +250,18 @@ def post(self):
 
                 all_event_objs.extend(event_objs)
 
+                if activity_data.name in ('consent','withdrawal','deactivate'):
+                    data = {
+                        'event_type': activity_data.name,
+                        'participant_id': participant_id,
+                        'event_authored_time': event_objs[0]['event_authored_time']
+                    }
+                    if GAE_PROJECT == 'localhost':
+                        pass
+                    else:
+                        _task = GCPCloudTask()
+                        _task.execute(endpoint='update_participant_summary_task', payload=data, queue='nph')
```

Comment:
This needs to be `update_participant_summary_for_nph_task` based on the route implementation.